js/main.js

```diff
@@ -205,8 +205,35 @@
 			$('.telescopio').css('display','block');

```

Comment:
I know this exceed this PR, but we should think about replace these spanish names using english.